src/cryptsetup/cryptsetup-generator.c

```diff
@@ -396,19 +396,23 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 r = sscanf(value, "%m[0-9a-fA-F-]=%ms", &uuid, &uuid_value);
```

Comment:
So `uuid_value` is not used any more and this code is partially unused. Why not use something like `%m[0-9a-fA-F-]=%n` to get the pointer to the beginnning of the keyspec.